products/sle/main.pm

```diff
@@ -443,7 +443,12 @@ sub load_boot_tests() {
         }
     }
     elsif (uses_qa_net_hardware) {
-        loadtest "installation/qa_net.pm";
+        if (get_var("BEIJING")) {
```

Comment:
I see no problem with the variable. The bug is actually the name of qa_net.pm, which hardcodes Nürnberg as center of the world.